app/views/traces/_page.html.erb

```diff
@@ -0,0 +1,19 @@
+<turbo-frame id="pagination" target="_top">
+  <%= render "shared/pagination",
+             :newer_key => "traces.page.newer",
+             :older_key => "traces.page.older",
+             :newer_id => @newer_traces_id,
+             :older_id => @older_traces_id %>
+
+  <table id="trace_list" class="table table-borderless table-striped">
+    <tbody>
+      <%= render @traces %>
+    </tbody>
+  </table>
+
+  <%= render "shared/pagination",
+             :newer_key => "traces.page.newer",
+             :older_key => "traces.page.older",
+             :newer_id => @newer_traces_id,
+             :older_id => @older_traces_id %>
+</turbo-frame id="pagination">
```

Comment:
Closing tag doesn't need `id`. Instead of `</turbo-frame id="pagination">` should be `</turbo-frame>`

Reply:
Thanks for spotting that - it should be fixed now.